### ISSUE TITLE
Distinguish "Nothing in here. Upload something!" and "Folder not synchronized"

### ISSUE THREAD

**Issue by LukeOwncloud** [owncloud/android#1257]
After installing oC app version 1.8, no content is yet synced. When browsing into a folder without Internet connection it shows: "Nothing in here. Upload something!" (file_list_empty)
This is usually wrong. Instead it should say something like "Folder not synchronized" if the folder was never synchronized, yet.

**Comment by elpraga** [comment]
I just checked it out and You actually don't even know when the server cannot be reached, 

I stopped the server and tried to update the directory, it was showing the indication that it usually shows when updating and after a while it just stopped letting me believe that the update has concluded.

Shouldn't a user get some kind of warning like: 

_Server couldn't be reached now, only local file available. Try again later_ or something similar to let us know we are not synced for now?

**Comment by LukeOwncloud** [comment]
Yep, that is also a problem I already reported here: https://github.com/owncloud/android/issues/688
